steps/compliance/mrn-date/MRNDate.js

```diff
@@ -12,11 +12,13 @@ class MRNDate extends Question {
     }
 
     get form() {
-
         return form(
-            field('day').validate(regex(numbers, this.content.fields.day)),
-            field('month').validate(regex(numbers, this.content.fields.month)),
-            field('year').validate(regex(numbers, this.content.fields.year)),
+            field('day')
+                .joi(this.content.fields.day.error.required, Joi.string().regex(numbers).required()),
```

Comment:
Fairly sure the `Joi.string()` part is unnecessary btw. At the moment, all fields values are strings. We haven't got in to custom parsing and more complex field types yet but I'll make sure that the default for a field is always string.

Reply:
I'm not sure I follow. When creating a Joi regex schema that's how it's defined, as shown in the example here: https://github.com/hapijs/joi. If I removed the `Joi.string()` part as suggested what would the schema look like?

Reply:
I had been thinking just `Joi.regex()` but ignore me, the regex function only exists on `Joi.string()`.